core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2788,15 +2788,26 @@ public static double log(BigDecimal d0, BigDecimal d1) {
     return Math.log(d0.doubleValue()) / Math.log(d1.doubleValue());
   }
 
-  /** SQL {@code LOG2(number)} function applied to double values. */
-  public static @Nullable Double log2(double number) {
-    return (number <= 0) ? null : log(number, 2);
+  /** SQL {@code LOGMySqlSpark(number, number2)} function applied to double values. */
```

Comment:
Just call it LOG and mention it's for those libraries. Your comment it makes it seem like `LOGMySqlSpark` is a function.

Reply:
done